docs/src/assets/js/search.js

```diff
@@ -114,6 +114,25 @@ function maintainScrollVisibility(activeElement, scrollParent) {
 
 }
 
+/**
+ * Debounces the provided callback with a given delay.
+ * @param {Function} callback The callback that needs to be debounced.
+ * @param {Number} delay Time in ms that the timer shoud wait before the callback is executed.
+ * @returns {Function}
+ */
+function debounce(callback, delay) {
+    let timer;
+    return (...args) => {
+        if (timer) clearTimeout(timer)
+        timer = setTimeout(() => callback(...args), delay)
```

Comment:
```suggestion
        timer = setTimeout(() => callback.apply(this, args), delay)
```
Use the apply method to bind the `this` & arguments to the called function